src/utils.ts

```diff
@@ -81,6 +81,7 @@ export const SETTINGS_DEFAULT_HOSTNAME = 'localhost';
 export const SETTINGS_DEFAULT_PORT = 9222;
 export const SETTINGS_DEFAULT_URL = path.resolve(path.join(__dirname, 'startpage', 'index.html'));
 export const SETTINGS_WEBVIEW_NAME = 'Edge DevTools';
+export const SETTINGS_SCREENCAST_WEBVIEW_NAME = 'Edge DevTools';
```

Comment:
Not sure the screencast title should be the same as the Devtools? Can we add "preview" or "screencast" to the title?

Reply:
I changed it because I got confused by the different names :), usually there is 1:1 association between tabs and extension so having "Edge Devtools Preview" I thought I had a different extension installed (from my own experiments on the hackaton). I don't have a strong opinion for naming so I'm ok either way. I'll bring this to the pod, to hear everyone's thoughts but open to anything :)

Reply:
I'd personally prefer for them to have separate names. If I have both tabs open, I want to be able to see at a glance which is which if neither are focused. I'd be fine with "Edge Devtools" and "Edge Devtools: Screencast" or something like that